src/dbos-runtime/runtime.ts

```diff
@@ -59,16 +71,11 @@ export class DBOSRuntime {
     return classes;
   }
 
-  /**
-   * Start an HTTP server hosting an application's functions.
-   */
-  startServer() {
-    // CLI takes precedence over config file, which takes precedence over default config.
-
-    const server: DBOSHttpServer = new DBOSHttpServer(this.dbosExec)
-
-    this.server = server.listen(this.runtimeConfig.port);
-    this.dbosExec.logRegisteredHTTPUrls();
+  onSigterm(): void {
+    this.dbosExec?.logger.info("Stopping executor: received a termination signal");
```

Comment:
```suggestion
    this.dbosExec?.logger.info("Stopping application: received a termination signal");
```

We are not stopping the execution environment but the DBOS application